src/desktop/apps/auction/queries/filter.js

```diff
@@ -24,7 +24,7 @@ export const filterQuery = `
       aggregations {
         slice
         counts {
-          value
+          id: value
```

Comment:
The `Checkbox` component is looking at `id` instead of `value` so that was the source of the bug inside of `ArtistFilter.js`.